superset-frontend/src/explore/controlUtils/standardizedFormData.test.tsx

```diff
@@ -0,0 +1,267 @@
+      expect(formData[key]).toEqual(value);
+    });
+  });
+});
+
+describe('should transform form_data between table and bigNumberTotal', () => {
+  const tableVizFormData = {
+    datasource: '30__table',
+    viz_type: 'table',
+    time_grain_sqla: 'P1D',
+    time_range: 'No filter',
+    query_mode: 'aggregate',
+    groupby: ['name'],
+    metrics: ['count'],
+    all_columns: [],
+    percent_metrics: [],
+    adhoc_filters: [],
+    order_by_cols: [],
+    row_limit: 10000,
+    server_page_length: 10,
+    order_desc: true,
+    table_timestamp_format: 'smart_date',
+    show_cell_bars: true,
+    color_pn: true,
+    applied_time_extras: {},
+    url_params: {
+      form_data_key:
+        'p3No_sqDW7k-kMTzlBPAPd9vwp1IXTf6stbyzjlrPPa0ninvdYUUiMC6F1iKit3Y',
+      dataset_id: '30',
+    },
+  };
+  const tableVizStore = {
+    form_data: tableVizFormData,
+    controls: {
+      datasource: {
+        value: '30__table',
+      },
+      viz_type: {
+        value: 'table',
+      },
+      slice_id: {},
+      cache_timeout: {},
+      url_params: {
+        value: {
+          form_data_key:
+            'p3No_sqDW7k-kMTzlBPAPd9vwp1IXTf6stbyzjlrPPa0ninvdYUUiMC6F1iKit3Y',
+          dataset_id: '30',
+        },
+      },
+      granularity_sqla: {},
+      time_grain_sqla: {
+        value: 'P1D',
+      },
+      time_range: {
+        value: 'No filter',
+      },
+      query_mode: {
+        value: 'aggregate',
+      },
+      groupby: {
+        value: ['name'],
+      },
+      metrics: {
+        value: ['count'],
+      },
+      all_columns: {
+        value: [],
+      },
+      percent_metrics: {
+        value: [],
+      },
+      adhoc_filters: {
+        value: [],
+      },
+      timeseries_limit_metric: {},
+      order_by_cols: {
+        value: [],
+      },
+      server_pagination: {},
+      row_limit: {
+        value: 10000,
+      },
+      server_page_length: {
+        value: 10,
+      },
+      include_time: {},
+      order_desc: {
+        value: true,
+      },
+      show_totals: {},
+      emit_filter: {},
+      table_timestamp_format: {
+        value: 'smart_date',
+      },
+      page_length: {},
+      include_search: {},
+      show_cell_bars: {
+        value: true,
+      },
+      align_pn: {},
+      color_pn: {
+        value: true,
+      },
+      column_config: {},
+      conditional_formatting: {},
+    },
+    datasource: {
+      type: 'table',
+      columns: [],
+    },
+  };
+
+  beforeAll(() => {
+    getChartControlPanelRegistry().registerValue(
+      'big_number_total',
+      new BigNumberTotalChartPlugin().controlPanel,
+    );
+    getChartControlPanelRegistry().registerValue(
+      'table',
+      new TableChartPlugin().controlPanel,
+    );
+  });
+
+  test('transform', () => {
+    // table -> bigNumberTotal
+    const sfd = new StandardizedFormData(tableVizFormData);
+    const { formData: bntFormData, controlsState: btnControlsState } =
```

Comment:
I assume this is a typo:
```suggestion
    const { formData: bntFormData, controlsState: bntControlsState } =
```

Reply:
Nice catch! it's a typo.